commands/certmanager/certificates.go

```diff
@@ -0,0 +1,156 @@
+package certmanager
+
+import (
+	"fmt"
+
+	"github.com/fatih/structs"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	"github.com/ionos-cloud/ionosctl/pkg/printer"
+	ionoscloud "github.com/ionos-cloud/sdk-go-cert-manager"
+)
+
+func CertCmd() *core.Command {
+	certCmd := &core.Command{
+		Command: &cobra.Command{
+			Use:              "certificate-manager",
+			Short:            "Certificate Manager Operations",
+			Long:             "The sub-commands of `ionosctl certificate-manager` allows you to manage the SSL Certificates under your account.",
+			TraverseChildren: true,
+		},
+	}
+	certCmd.AddCommand(CertGetCmd())
+	certCmd.AddCommand(CertPostCmd())
+	certCmd.AddCommand(CertListCmd())
+	certCmd.AddCommand(CertDeleteCmd())
+	certCmd.AddCommand(CertGetApiVersionCmd())
+	certCmd.AddCommand(CertPatchCmd())
+	return certCmd
+}
+
+func getCertPrint(resp *ionoscloud.APIResponse, c *core.CommandConfig, cert *[]ionoscloud.CertificateDto) printer.Result {
+	r := printer.Result{}
+	if c != nil {
+		if resp != nil {
+			r.Resource = c.Resource
+			r.Verb = c.Verb
+			r.WaitForState = viper.GetBool(core.GetFlagName(c.NS, ArgWaitForState)) // this boolean is duplicated everywhere just to do an append of `& wait` to a verbose message
+		}
+		if cert != nil {
+			r.OutputJSON = cert
+			r.KeyValue = getCertRows(cert)                                                    // map header -> rows
+			r.Columns = getCertHeaders(viper.GetStringSlice(core.GetFlagName(c.NS, ArgCols))) // headers
+		}
+	}
+	return r
+}
+
+func printProperties(value *ionoscloud.CertificateDto, c *core.CommandConfig, flags []bool) string {
+	var printString string
+	if flags[0] {
+		printString += *value.Properties.Certificate
+	}
+	if flags[1] {
+		printString += *value.Properties.CertificateChain
+	}
+	return printString
+}
```

Comment:
1. Don't concatenate these into a single string. return either one or the other (If both are set, it's ok, print one of them, we can mark them as exclusive flags in the commands)
2. instead of taking a `[]bool` it would make the code easier to understand if we take two bools, `isSetPrintCert`, `isSetPrintChain` or whatever better naming comes to mind
3. Is this func used anywhere other than in the `get` cmd? if not, we can just move the functionality there and no need to use a func. Otherwise, fi it's used in multiple places, a func is nice, it helps reduce code duplication. Nice!

Reply:
This function prints only one option.